rs/consensus/src/dkg/payload_builder.rs

```diff
@@ -1206,9 +1209,13 @@ mod tests {
                 assert_eq!(conf.max_corrupt_dealers().get(), 2);
                 assert_eq!(
                     conf.threshold().get().get(),
-                    match *tag {
+                    match tag {
                         NiDkgTag::LowThreshold => 3,
                         NiDkgTag::HighThreshold => 5,
+                        /////////////////////////////////////////////////////
+                        // TODO(CON-1417): extend this test once we have support for vetKD transcripts in registry CUPs
+                        /////////////////////////////////////////////////////
+                        NiDkgTag::HighThresholdForKey(_) => panic!("not applicable"),
```

Comment:
Maybe instead of panicking with the not very informative `"not applicable"`, we use `unimplemented("CON-1417")`. I mean, if we were to ever hit this case (which I don't believe we will), we would know much faster whats going on

Reply:
Good point: Done in https://github.com/dfinity/ic/pull/2445/commits/964dbb5130d7a770106d15f28c647007d639a2f1

I changed it to `todo!("CON-1417")`, because `unimplemented` means we don't intend to ever implement this, while `todo!` means we haven't implemented it yet.

Same for one other panic.

Reply:
Resolving this comment, because it is addressed (see above). Let me know if you have further comments.